content/patterns/radio/examples/radio-activedescendant.html

```diff
@@ -25,128 +25,120 @@
-          <li><a href="radio-rating.html">Rating Radio Group Example</a>: Radio group that provides input for a five-star rating scale.</li>
+          <li><a href="radio.html">「tabindex の移動 (roving tabindex)」を使用したラジオグループの例</a>: キーボードフォーカスを管理するために「tabindex の移動 (roving <code>tabindex</code>)」を使用するラジオボタングループ。</li>
+          <li><a href="radio-rating.html">評価用のラジオグループの例</a>: 五つ星の評価スケールに入力を提供するラジオグループ。</li>
         </ul>
       </section>
 
       <section>
         <div class="example-header">
-          <h2 id="ex_label">Example</h2>
+          <h2 id="ex_label">例</h2>
         </div>
         <div role="separator" id="ex_start_sep" aria-labelledby="ex_start_sep ex_label" aria-label="Start of"></div>
         <div id="ex1">
-          <h3 id="group_label_1">Pizza Crust</h3>
+          <h3 id="group_label_1">ピザのクラスト</h3>
 
           <ul class="radiogroup-activedescendant" role="radiogroup" aria-labelledby="group_label_1" aria-activedescendant="rb11" tabindex="0">
-            <li id="rb11" role="radio" aria-checked="false">Regular crust</li>
-            <li id="rb12" role="radio" aria-checked="false">Deep dish</li>
-            <li id="rb13" role="radio" aria-checked="false">Thin crust</li>
+            <li id="rb11" role="radio" aria-checked="false">レギュラークラスト</li>
+            <li id="rb12" role="radio" aria-checked="false">ディープディッシュ</li>
+            <li id="rb13" role="radio" aria-checked="false">シンクラスト</li>
           </ul>
 
-          <h3 id="group_label_2">Pizza Delivery</h3>
+          <h3 id="group_label_2">ピザのお受け渡し</h3>
 
           <ul class="radiogroup-activedescendant" role="radiogroup" aria-labelledby="group_label_2" aria-activedescendant="rb21" tabindex="0">
-            <li id="rb21" role="radio" aria-checked="false">Pick up</li>
-            <li id="rb22" role="radio" aria-checked="false">Home Delivery</li>
-            <li id="rb23" role="radio" aria-checked="false">Dine in Restaurant</li>
+            <li id="rb21" role="radio" aria-checked="false">お持ち帰り</li>
+            <li id="rb22" role="radio" aria-checked="false">宅配</li>
+            <li id="rb23" role="radio" aria-checked="false">イートイン</li>
           </ul>
         </div>
         <div role="separator" id="ex_end_sep" aria-labelledby="ex_end_sep ex_label" aria-label="End of"></div>
       </section>
 
       <section>
-        <h2>Accessibility Features</h2>
+        <h2>アクセシビリティ機能</h2>
         <ul>
           <li>
-            The radio button referenced by <code>aria-activedescendant</code> is scrolled into view when it is not visible in the graphical rendering.
-            This can occur under many conditions, but is most common when people with visual impairments use a browser's zoom feature to increase the size of content.
+            <code>aria-activedescendant</code> で参照されるラジオボタンは、グラフィカルなレンダリングで視覚的に見えない場合に、スクロールによって見えるようになります。これは多くの状況で生じる可能性がありますが、もっともよくあるのが、視覚障害のある人がブラウザのズーム機能を使用してコンテンツのサイズを拡大している場合です。
```

Comment:
「グラフィカルなレンダリングで視覚的に見えない場合に」は少し回りくどい表現になってしまっているかなと感じました。また「scroll into view」のviewはおそらくビューポートのことを指していると思われ、「見えるようになる」と訳すのは少し違和感がありました。

少し意訳気味になっていますが以下のような感じではどうでしょうか？

```suggestion
            <code>aria-activedescendant</code> で参照されるラジオボタンが画面に表示されていない場合、表示範囲にスクロールされます。これは多くの状況で生じる可能性がありますが、もっともよくあるのが、視覚障害のある人がブラウザのズーム機能を使用してコンテンツのサイズを拡大している場合です。
```

Reply:
異論ありません！